### PR TITLE
Delete 'P2P msg exchange' row from table

### PR DESCRIPTION
**Pull request by markmuir87**
There are a number of reasons why this line should be deleted (or at least heavily expanded on and qualified).
- "P2P message exchange" is the wrong semantics, based on what I think you're getting at here. P2P message exchange can be done in any old way. For instance, I can send a JSON mesage to my friend John using good ol' HTTP (and implicitly all lower level protocols in the stack). That's a peer-to-peer message exchange. What I think we're really talking about here are decentralised routing protocols (however please correct me if I'm wrong).
- Assuming we are talking about DRPs, I can't see why we would suggest ebMS3/AS4 with SAML 2.0 as the 'preferred technology'. This choice violates a number of principles set out by the API design guide (e.g. simplicity), and also violates the principle of the section it is in: Ubiquitous Standards. Just because the Commonwealth is using it for SuperStream, doesn't make it ubiquitous. 
- This also raises the question of 'how decentralised' do we mean?
- At a glance, there are at least two _far_ more ubiquitous and simpler 'decentralised' protocols in common use: [Kademlia](https://en.wikipedia.org/wiki/Kademlia) or the [BitTorrent protocol/Mainline DHT](https://en.wikipedia.org/wiki/Mainline_DHT). And depending on the level of decentralisation, you could easily add DNS (federated), Tor (partial-decent) or [HWMP in IEEE 802.11s](https://en.wikipedia.org/wiki/IEEE_802.11s#Routing_protocols) or [B.A.T.M.A.N](https://en.wikipedia.org/wiki/B.A.T.M.A.N.) (fully-decent).
- It's also unclear what role (if any) government would/should play in a DRP-based system.

In short, this is a much larger issue that requires much more detail and qualification. And regardless of what kind of DRP we're specifically talking about, ebMS3/AS4 shouldn't even be one of the contenders given its complexity and lack of ubiquity.

Thoughts @monkeypants, @markmckenzie and @nokout?

### CODE REVIEW

**Comment by markmuir87** [comment]
Just to add on ebMS3/AS4 (and this is part of a more general issue):

* I don't think the government should endorse 'open standards' published under [ambiguous licensing and/or copyright terms](https://en.wikipedia.org/wiki/OASIS_(organization)#Patent_disclosure_controversy). If the government endorses or mandates a standard that turns out to require a standard's compliant implementation to infringe on third-party patents, there's a risk the holder of that patent or copyright can 'tax' (via licensing fees) their retail competitors. This would effectively be an indirect tax on all end-consumers in that API market (which could potentially be a large proportion of the Australian population if the API relates to, say, income tax or BAS statements).

* For instance, here are extracts from the copyright notice for [ebms3/AS4](http://docs.oasis-open.org/ebxml-msg/ebms/v3.0/profiles/AS4-profile/v1.0/os/AS4-profile-v1.0-os.html) (emphasis mine):

> OASIS **requests** that any OASIS Party or any other party that believes it has patent claims that  would necessarily be infringed by implementations of this OASIS Committee Specification or OASIS  Standard, to notify OASIS TC Administrator and **provide an indication** of its willingness to grant  patent licenses to such patent claims in a manner consistent with the IPR Mode of the OASIS  Technical Committee that produced this specification.

> OASIS invites any party to contact the OASIS TC Administrator if it is aware of a claim of ownership of any patent claims that would necessarily be infringed by implementations of this specification by a patent holder that is not willing to provide a license to such patent claims in a manner consistent with the IPR Mode of the OASIS Technical Committee that produced this specification. OASIS **may include** such claims on its website, **but disclaims any obligation to do so**....

> ...**neither does [OASIS] represent that it has made any effort to identify any such rights**. 
Information on OASIS' procedures with respect to rights in any document or deliverable produced by  an OASIS Technical Committee can be found on the OASIS website. Copies of claims of rights made  available for publication and any assurances of licenses to be made available, or the result of an 
attempt made to obtain a general license or permission for the use of such proprietary rights by
implementers or users of this OASIS Committee Specification or OASIS Standard, can be obtained  
from the OASIS TC Administrator. **OASIS makes no representation that any information or list of 
intellectual property rights will at any time be complete**, or that any claims in such list are, in fact, 
Essential Claims.

On my reading, these terms mean:
* There is no **requirement** for patent holders, whose patents would be infringed by a standard compliant implementation, to make that fact known to anyone.
* If OASIS is made aware of 'potentially infringed patents', they do not guarantee they will publish this information (only that they 'may include' this information on their website), and in fact OASIS **explicitly disclaims** any obligation to do so.
* OASIS make no guarantee that any information regarding potentially 'infringed patents', if published at all, will be complete.

Although OASIS' ["intellectual property rights policy"](https://www.oasis-open.org/policies-guidelines/ipr#disclosure) places more stringent disclosure requirements on member organisations, and ebms3/as4 is licensed under their more permissive 'RF on Limited Terms' policy, the common terms of the licence are ambiguous. Additionally, the 'RF on Limited Terms' licence give 'obligated parties' (e.g. OASIS members) the right to "choice of law and dispute resolution". I can't be sure, but it sounds like this allows an 'OASIS licensor' to sue an Australian SWD in an Argentinian/Bulgarian/Martian court if there's a licensing dispute (or in whichever other jurisdiction has the most favourable laws for the patent holder).

Given the membership of OASIS is mostly huge multi-nationals like IBM and Microsoft (who have subsidiaries in pretty much every country), this kind of condition is worrying. 

I can also see a number of areas where the licensing terms might distort competition in downstream markets. For instance, although required to provide royalty-free licences under the 'RF on Limited Terms' mode, an 'obligated party' [e.g. OASIS member] can make this licence conditional on the licencee [e.g. a software developer] providing reciprocal licences for any of their IP that 'covers' the implemented standard. This potentially strangles any market competition and reduces incentive for non-OASIS SWDs to develop novel IP on top of OASIS standards (as they may be forced to hand it over to IBM or Microsoft in order to participate in the market).

Anyhow, this is just an instance of a wider issue which should be addressed in the guide (i.e. the IP status of endorsed 'open standards'). I'll raise an issue.
